assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py

```diff
@@ -45,18 +51,22 @@ def __init__(self, model_id: str):
         self._hf_api = HfApi(endpoint=self.HF_ENDPOINT)
         self._model_info = None
 
+    @retry(3)
+    def _fetch_model_info(self):
+        try:
+            model_list: List[ModelInfo] = self._hf_api.list_models(filter=ModelFilter(model_name=self._model_id))
+            for info in model_list:
+                if self._model_id == info.modelId:
+                    return info
+        except Exception as e:
+            raise ValueError(f"Failed to validate model id : {e}")
```

Comment:
Could we please use a custom error here? "InvalidINput" or the likes from AzureML Errors?

Reply:
Added. Please help to review

Reply:
Added. Please help to review